java/src/org/certificatetransparency/ctlog/serialization/Deserializer.java

```diff
@@ -168,18 +169,17 @@ public static TimestampedEntry parseTimestampedEntry(InputStream in) {
       int length = (int) readNumber(in, 3);
       signedEntry.x509 = readFixedLength(in, length);
     } else if (entryType == Ct.LogEntryType.PRECERT_ENTRY_VALUE) {
-      Ct.PreCert.Builder preCertBuilder = Ct.PreCert.newBuilder();
+      PreCert preCert = new PreCert();
 
-      byte[] arr = readFixedLength(in, 32);
-      preCertBuilder.setIssuerKeyHash(ByteString.copyFrom(arr));
+      preCert.issuerKeyHash = readFixedLength(in, 32);
 
       // set tbs certificate
-      arr = readFixedLength(in, 2);
+      byte[] arr = readFixedLength(in, 2);
```

Comment:
I'll be honest with you here, I don't really know, I only turned it from an assignment to a declaration, as I deleted the declaration that was at line 173 before.

I think it's indeed skipping 2 bytes, no idea why. I think it might work if I just call `readFixedLength` and ignore the return value (I'd add a comment to explain what's happening, of course!)? @eranmes should know.